modules/Tvpapi/resources/tvpapiAnalytics.js

```diff
@@ -127,7 +129,7 @@
 				"mediaType": 0,
 				"iMediaID": this.getProxyConfig('MediaID'),
 				"iFileID": this.fileId,
-				"iLocation": (this.getPlayer().isLive()&&!this.getPlayer().isDVR())?0:this.getCurrentTime()
+				"iLocation": (this.getPlayer().isLive() && !this.getPlayer().isDVR())?0:this.getCurrentTime()
```

Comment:
@chanakabr same here, extract the live and dvr to separate params.

Reply:
@OrenMe - done.